### PR TITLE
Adding existing cluster only sample app

### PR DESCRIPTION
**Pull request by ShashiShekhar32**
## Purpose

* ...
Adding existing cluster only sample app

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[x] Other... Please describe: Adding sample app for existing cluster only option
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Review by jhmadhav — APPROVED** [review]

**Review by jhmadhav — APPROVED** [review]

File: samples/k8s-existing-cluster-only-sample/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+applicationName: com.contoso.azurevote.existing
```
Comment (RamyasreeChakka) [inline samples/k8s-existing-cluster-only-sample/manifest.yaml:1]: Is this file required in this sample folder?

File: samples/k8s-existing-cluster-only-sample/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+- yaml
+- json
+products:
+- azure-kubernetes-service
+- azure-marketplace
+---
+
+# Existing cluster only sample
+
+This folder contains samples that shows how ISV can use only existing cluster option in their create ui definition.
```
Comment (RamyasreeChakka) [inline samples/k8s-existing-cluster-only-sample/README.md]: Can you please change the description as below?
This sample illustrates how ISV can allow an application to deploy only on existing clusters. The CreateUIDefinition and ARM template files in this folder shows how to use only existing cluster option.

File: samples/k8s-existing-cluster-only-sample/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+## The resulting UI Definition will show up as follows:
+
+![Alt text](images/UI_Sample.PNG)
+
+# Explanation
+
+For setting the scope to the existing cluster, create ui definition should store id of the existing cluster in output variable as follows
+
+## UIDefinition
+The following section creates an array of key value pairs
```
Comment (RamyasreeChakka) [inline samples/k8s-existing-cluster-only-sample/README.md]: Do we need this statement? Can this be replaced by text 'For setting the scope to the existing cluster, create ui definition should store id of the existing cluster in output variable as follows'

File: samples/k8s-existing-cluster-only-sample/README.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+- azure-marketplace
+---
+
+# Existing cluster only sample
+
+This folder contains samples that shows how ISV can use only existing cluster option in their create ui definition.
+
+## The resulting UI Definition will show up as follows:
+
+![Alt text](images/UI_Sample.PNG)
```
Comment (RamyasreeChakka) [inline samples/k8s-existing-cluster-only-sample/README.md:17]: Can you also add an example of Cluster Details image? Above image has only basic settings.

File: samples/k8s-existing-cluster-only-sample/README.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+page_type: sample
+languages:
+- yaml
+- json
+products:
+- azure-kubernetes-service
+- azure-marketplace
+---
+
+# Existing cluster only sample
```
Comment (RamyasreeChakka) [inline samples/k8s-existing-cluster-only-sample/README.md:11]: To improve discoverability, can you please include this sample in the 2nd row in the table in repo's readme.md https://github.com/Azure-Samples/kubernetes-offer-samples#samples-in-this-repo ?

File: samples/k8s-existing-cluster-only-sample/manifest.yaml (hunk 1 above)
Comment (ShashiShekhar32) [reply samples/k8s-existing-cluster-only-sample/manifest.yaml:1]: may be  yes, to depict the changes are only in those files. I see other samples also has the same so keeping that for consistency.

File: README.md [hunk 6]
```diff
@@ -16,6 +16,7 @@ This repository contains samples for creating Kubernetes Applications in Azure M
 | Sample | Description |
 |--------|-------------|
 |[k8s-offer-azure-vote](samples/k8s-offer-azure-vote/) | The Azure Vote application is a simple sample used throughout the Kubernetes Applications Marketplace documentation. The application consists of two pods, one running a flask web form, and the second a redis instance for temporary data storage. |
+|[k8s-existing-cluster-only-sample](samples/k8s-existing-cluster-only-sample/) | This sample shows how ISV can use only existing cluster option in their create ui definition. |
```
Comment (RamyasreeChakka) [inline README.md]: >This sample shows how ISV can use only existing cluster option in their create ui definition. [](http://example.com/codeflow?start=81&length=93)

nit: Can you please change this text to below?

This sample illustrates how ISV can allow an application to deploy only on existing clusters.

**Review by RamyasreeChakka — APPROVED** [review]

**Review by RamyasreeChakka — APPROVED** [review]
:shipit:
